Sources/PaymentButtons/CardButton.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import SwiftUI
+
+/// Configuration for Card button
+public final class CardButton: PaymentButton {
+
+    /// Available colors for CardButton.
+    public enum Color: String {
+        case gold
+        case white
+        case black
+        case silver
+        case blue
```

Comment:
This button will only ever be black so we can remove the other options.

Reply:
I lied, we also support a white BCDC button. I'll find the black card icon for this.